website/src/word-panel.css.tsx

```diff
@@ -0,0 +1,32 @@
+import { style } from "@vanilla-extract/css"
+import sprinkles from "./sprinkles.css"
+
+export const wholePanel = style({
+    position: "sticky",
+    top: 150,
+})
+
+export const wordPanelButton = style({
+    lineHeight: 1.5,
+    boxSizing: "border-box",
+    border: "transparent",
+    cursor: "pointer",
+    fontSize: "16px",
```

Comment:
Use a relative [font size in `rem` units](https://www.sitepoint.com/understanding-and-using-rem-units-in-css/), for example this might just be `1.1rem` if you want bigger than normal body text or `0.9rem` if you want a bit smaller.